MAVProxy/modules/mavproxy_chat/chat_voice_to_text.py

```diff
@@ -55,15 +66,25 @@ def record_audio(self):
 
         # calculate time recording should stop
         curr_time = time.time()
-        time_stop = curr_time + 5
+        time_stop = curr_time + 3
 
         # record until specified time
         frames = []
-        while curr_time < time_stop:
+
+        # logic for recording sound until someone is speaking.
+        isSpeaking = True
```

Comment:
Hi, I think our normal style is to use underscores between variables so let's change "isSpeaking" to "is_speaking".

Also maybe change the comment to be "record sound while user is speaking"